docs/source/tutorials.rst

```diff
@@ -1,21 +1,36 @@
+.. _tutorials:
+
 Tutorials
-###########
+######################
 
-.. _tutorials:
 
 
 Here a set of examples on how to use different MyoSuite models and non-stationarities.
-Jupyter-Notebooks can be found `here <https://github.com/facebookresearch/myosuite/tree/main/docs/source/tutorials>`__
+It is highly recommended to read through the `OpenAI Gym API <https://gymnasium.farama.org/>`__ before you start.
```

Comment:
@raku-slyu "It is highly recommended to read through the `OpenAI Gym API <https://gymnasium.farama.org/>`__ to get familiar with the Gym API"

Reply:
updated on commit : [https://github.com/MyoHub/myosuite/pull/192/commits/03b3d50913cf1751d5ab4f855aca0009828d40a2](url)